chyt/README.md

```diff
@@ -0,0 +1,14 @@
+#### CHYT powered by ClickHouse
+
+1.  Install YTsaurus cluster. Visit [YTsaurus Getting started webpage](https://ytsaurus.tech/docs/en/overview/try-yt)
```

Comment:
As far as I understand, the code is open-source, right? Script `benchmark.sh` ideally does as much setup as possible automatically, i.e. no user intervention. For examples how to do that, please see clickhouse/benchmark.sh, postgresql/benchmark.sh and duckdb/benchmark.sh.

Reply:
Yes, YTsaurus is open-source system. But CHYT is a small part of it. `Benchmark.sh` uses pre-installed cluster with default clique to do benchmark test.
YTsaurus cluster can be installed, for example, using [k8s operator](https://github.com/ytsaurus/ytsaurus-k8s-operator)
All possible variants are described in [documentation](https://ytsaurus.tech/docs/en/overview/try-yt).

Reply:
As I mentioned, we need to reduce the variability here ... As someone who wants to verify the benchmark results, I like to run `benchmark.sh` and have it install everything by itself. The only think that I would be able to choose is the hardware the system runs on.

Reply:
We have a possibility to create a [demo cluster](https://ytsaurus.tech/docs/en/overview/try-yt#demo-stand) for everyone who wants to try YTsaurus. Would it be ok to verify the results?

Reply:
One installation option is [Docker](https://ytsaurus.tech/docs/en/overview/try-yt#using-docker). That seems the alternative with the least amount of complexity and the best reproducibility (compared to k8s and the demo cluster).

My preference would be if benchmark.sh sets up the docker container, does other preparations, and then runs the measurements.

Reply:
Deploying a cluster with 360 vCPUs and 720 GB of RAM using a single VM can be quite challenging. In our case, we use a Kubernetes cluster with nodes of the type c6a.8xlarge and network SSDs that perform similarly to gp2 volumes.
We also aim to demonstrate various cluster sizes, not just the smallest one. If necessary, we can remove the "serverless" tag and instead specify the number of CHYT instances in the configuration.
Given the large size of our cluster, Docker deployment was not utilized for benchmarking, as it may yield different results. The easiest way to reproduce our results is by booking a demo cluster through our website.
Additionally, I can include a step-by-step guide in the README.md file to assist with the setup.

Reply:
Okay, let's add a step by step guide in the README, I'll afterwards try my best to reproduce, then I will merge.